content/docs/gto/user-guide/dvc.md

```diff
@@ -0,0 +1,144 @@
+# GTO with DVC
+
+In many ML projects, data isn't stored in a Git repository and needs to be
+downloaded from external sources. [DVC](https://dvc.org) is a common way to
+store binaries for the artifacts registered with GTO.
+
+<details>
+
+### Learn about different approaches to this
+
+1. You can commit artifacts to Git repo. If they aren't small enough, this is
+   not recommended. To bypass this limitation, you can use
+   [Git-lfs](https://git-lfs.github.com).
+2. You can version binaries with [DVC](https://dvc.org/) and commit pointers to
+   them to the repo. This is the recommended approach for large files.
+3. You can version binaries manually somewhere, specifying URL to them as `path`
+   in `artifacts.yaml`. This can be done, if your artifacts are already
+   versioned by some external system.
```

Comment:
Many Qs:

- Why is this hidden?
- `You can commit artifacts to Git repo` - is that a realistic scenario? Let's not even mention it otherwise.
- Mention DVC before Git LFS.
- Why are we talking about data versioning? Seems to me it complicates the explanation.
- Are artifacts always "binaries" ? Maybe just call them "artifact files", "large files", "data", or something like that (more generic).
- `versioned by some external system` - what are we referring to here? (Examples)

Reply:
Kk 🙂